lib/profile/save_profile/sections/education/education_item.dart

```diff
@@ -66,16 +71,32 @@ class _SaveEducationItemState extends State<SaveEducationItem> {
   Widget build(BuildContext context) {
     store = StoreProvider.of<AppState>(context);
 
+    screenWidth = MediaQuery.of(context).size.width;
+    screenHeight = MediaQuery.of(context).size.height;
+    
+    isSchoolNameInvalid = false;
+    isDegreeInvalid = false;
+    isSchoolNameFocused = false;
+    isDegreeFocused = false;
+
     return GestureDetector(
       onTap: unfocusFields,
       child: Scaffold(
-        appBar: AppBarWithSave(
-          appBarTitle: widget.editMode ? editEducation : addEducation,
-          formKey: _saveEducationItemKey,
-          closeActionEnabled: true,
-          onCloseActionTap: handleClose,
-          onSaveActionTap: handleSaveEducation,
-        ),
+        appBar: AppBar(
+            elevation: 0.0,
+            leading: IconButton(
+            iconSize: screenHeight * 0.02,
+            onPressed: () {
+              handleClose();
+              Navigator.pop(context);
+              },
```

Comment:
Take `Navigator.pop(context)` out here and add it in the `handleClose()` function instead

Reply:
Got ya

Reply:
Also, fix indentation here with the `onPressed`!